.github/actions/tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+CONTAINER=${PROJECT}_api.1.$(docker service ps -f "name=${PROJECT}_api.1" ${PROJECT}_api -q --no-trunc | head -n1)
+docker exec -i ${CONTAINER} bundle exec rake test ${TEST_ENV} ${OPTIONS}
```

Comment:
what is this env var `TEST_ENV`?

Reply:
This comes from the old script, I remove it.